llvm/utils/repo/repo.cmake

```diff
@@ -24,10 +24,57 @@ else ()
     set (utils_dir /usr/share/repo)
 endif ()
 
-SET (CMAKE_C_FLAGS_DEBUG "${CMAKE_C_FLAGS_DEBUG} -O0 -fno-exceptions -fno-rtti" CACHE STRING "Default C Flags Debug")
-SET (CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} -O0 -fno-exceptions -fno-rtti" CACHE STRING "Default CXX Flags Debug")
-SET (CMAKE_C_FLAGS_RELEASE "${CMAKE_C_FLAGS_RELEASE} -O3 -fno-exceptions -fno-rtti" CACHE STRING "Default C Flags Release")
-SET (CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} -O3 -fno-exceptions -fno-rtti" CACHE STRING "Default CXX Flags Release")
+# The user may specify the libcxxabi_include by giving
+# '-Dlibcxxabi_include:STRING=/path/to/clang/include' on the command line
+if (libcxxabi_include)
+	set (libcxxabi_include "${libcxxabi_include}")
+endif ()
+
+# The user may specify the exception and rtti by giving
+# '-Dexception_rtti:BOOL=Yes' on the command line
+if (exceptions_rtti)
+	set (exceptions_rtti "-fexceptions -frtti")
+else ()
+	set (exceptions_rtti "-fno-exceptions -fno-rtti")
+endif ()
+
+# The user may specify the musl_install by giving
```

Comment:
It would be more useful to describe what this variable means rather than just that I can set it. Presumably it refers to the directory in which musl-c is installed.

Reply:
Fixed in the commit https://github.com/SNSystems/llvm-project-prepo/pull/145/commits/6c4f10ca7cb881e1330eb14bea461620683715d0.

Reply:
> musl_install: the path of the directory in which musl libc is installed\

This should be in the future tense, I think: “the path of the directory in which musl libc will be installed". We’re describing the path which will be used by a later install step, aren’t we? It’s not a path that we expect to already exist.

Reply:
This file is repo.cmake file. We except both the `musl_install` and `llvm_install` already exist/installed and then we can use musl libc and LLVM libraries to build the user's project. Therefore, the `musl_install` and `llvm_install` should not be future tense.